CMakeLists.txt

```
@@ -11,6 +11,9 @@ cmake_minimum_required(VERSION 2.8.11)
 project(libLAS)
 
 
+
+set(CMAKE_CXX_STANDARD 14)
```

Thanks for the contribution but in my opinion this line should read
```suggestion
set(CMAKE_CXX_STANDARD 14 CACHE STRING "C++ standard version to use (default is 14)")
```